inlang/source-code/website/src/services/local-storage/src/LocalStorageProvider.tsx

```diff
@@ -146,6 +147,8 @@ export function LocalStorageProvider(props: { children: JSXElement }) {
 		}
 	}
 
+	window.addEventListener("storage", onStorageSetByOtherWindow)
```

Comment:
I'm not sure if it is this part of the code, but something in the LocalStorageProvider triggers now a "window is not defined" error on the website.

Reply:
@NiklasBuchfink  is there ssr?

Reply:
i need a screenshot where and how that happens